tasks/main.yml

```diff
@@ -48,6 +48,15 @@
     state: present
     filename: koha-secondary
 
+- name: Configure pinning for koha-common
+  template:
+    src: pin-koha.j2
+    dest: /etc/apt/preferences.d/koha
+    owner: root
+    group: root
+    mode: u=rw,g=r
+  when: koha_secondary_package_rep_domain is defined and koha_secondary_package_rep_domain|length > 0
```

Comment:
is the length > 0 needed? Would

`when: koha_secondary_package_rep_domain is defined and koha_secondary_package_rep_domain`

be equivalent? Please add documentation also for this variable to the README file.

Reply:
length > 0 and just variable name has a bit different semantics so let's go with length > 0 given it works for "0" string too.